spring-core/src/test/java/org/springframework/util/ReflectionUtilsTests.java

```diff
@@ -231,6 +232,8 @@ void findMethod() {
 		assertThat(ReflectionUtils.findMethod(B.class, "bar", String.class)).isNotNull();
 		assertThat(ReflectionUtils.findMethod(B.class, "foo", Integer.class)).isNotNull();
 		assertThat(ReflectionUtils.findMethod(B.class, "getClass")).isNotNull();
+		assertThat(ReflectionUtils.findMethod(B.class, "baz", String[].class).isBridge()).isFalse();
```

Comment:
Since there are no generics involved here, there is never a bridged method for `baz(String... args)`.